### PR TITLE
switch to using bion instead of ebin

### PR DESCRIPTION
**Pull request by zingale**
this is consistent with the aprox nets and eliminates a multiply on the sum over dY/dt

### CODE REVIEW

**Comment by zingale** [comment]
it looks like if I convert the ebin value in the old form to MeV / nucleus I get 127.62077 instead of 127.62093, but I am not sure where that first value comes from.  The latter value matches what is in aprox21.

File: networks/triple_alpha_plus_cago/actual_network.F90 [hunk 1]
```diff
@@ -28,28 +29,30 @@ module actual_network
...
 contains
 
   subroutine actual_network_init()
 
     call network_properties_init()
-    allocate(ebin(nspec))
+    allocate(bion(nspec))
 
 
     ! our convention is that binding energy is negative.  The following are
```
Comment (maxpkatz) [inline networks/triple_alpha_plus_cago/actual_network.F90:42]: This comment needs to be updated

File: networks/triple_alpha_plus_cago/actual_network_data.cpp [hunk 2]
```diff
@@ -20,10 +20,10 @@ void actual_network_init ()
     // the binding energies per unit mass (erg / g) obtained by converting
```
Comment (maxpkatz) [inline networks/triple_alpha_plus_cago/actual_network_data.cpp:20]: This comment needs to be updated

File: networks/triple_alpha_plus_cago/actual_network.F90 (hunk 1 above)
Comment (zingale) [reply networks/triple_alpha_plus_cago/actual_network.F90]: comment updated

File: networks/triple_alpha_plus_cago/actual_network_data.cpp (hunk 2 above)
Comment (zingale) [reply networks/triple_alpha_plus_cago/actual_network_data.cpp:20]: I've updated this

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2020-09-18-001/index.html

File: networks/triple_alpha_plus_cago/actual_network.F90 [hunk 3]
```diff
@@ -28,28 +29,30 @@ module actual_network
...
     ! the energies in MeV to erg then multiplying by (N_A / aion) where
     ! N_A = 6.0221415e23 is Avogadro's number
-    ebin(ihe4)  = -6.8253797e18_rt    !  28.39603 MeV / nucleon
-    ebin(ic12)  = -7.4103097e18_rt    !  92.16294 MeV / nucleon
-    ebin(io16)  = -7.6959581e18_rt    ! 127.62093 MeV / nucleon
-    ebin(ife56) = -8.4813001e18_rt    ! 492.25389 MeV / nucleon
+    bion(ihe4)  = 28.29603_rt ! MeV / nucleus
+    bion(ic12)  = 92.16294_rt ! MeV / nucleus
+    bion(io16)  = 127.62093_rt ! MeV / nucleus
+    bion(ife56) = 492.25389_rt ! MeV / nucleus
```
Comment (harpolea) [inline networks/triple_alpha_plus_cago/actual_network.F90:49]: aprox21 is using `bion(ife56) = 492.2450e0_rt`. Which value do we believe is correct?

Comment (maxpkatz) [reply networks/triple_alpha_plus_cago/actual_network.F90:49]: Let's sort that out after this PR. Eventually we should unify all of the physical constants across all the networks, but we don't have to tackle that as part of this code change.

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**